cmd/setup.go

```diff
@@ -51,6 +52,11 @@ func configureEnvironment(conf config) (err error) {
 		err = sponsor.ConfigureSponsorship(conf.SponsorToken)
 	}
 
+	// setup session log
+	if err == nil && conf.Database.Path != "" {
```

Comment:
Ich würde das nicht zu einem optionalen Feature machen sondern die DB in eine Standard-Location (`/usr/local/etc/evcc.sqlite`, `~/.evcc/data.sqlite`, ...) legen. Der Ort kann ja per config überschreibbar sein. Das machts zum einen in der Erstkonfiguration einfacher (eine Entscheidung weniger) und zum anderen können wir dann in den Guides (bspw. für Docker) einen konkreten Pfad angeben den man rausmappen muss um die Daten auch persistent zu halten.

Reply:
Hast Du eine Idee wie ein plattformunabhängiger Pfad aussehen würde?

Reply:
Finde, der Homeverzeichnis ist erstmal ein guter Default.
https://pkg.go.dev/os#UserHomeDir

Reply:
Ok, ist schon so :D